content/courses/calculus/integral/substitution/index.md

```diff
@@ -1,7 +1,11 @@
 
 ---
 title: Integration by Substitution
+index: 18
+type: lesson
```

Comment:
Remember that these are necessary for every single file!

Reply:
I don't know if this is a request or if it's just a comment. If it's a request, do I need to approve it?

Reply:
this  is just a reminder